packages/utils/src/createSchemaUtils.ts

```diff
@@ -27,26 +27,30 @@ import {
   toIdSchema,
   toPathSchema,
 } from './schema';
+import { DefaultFormStateBehavior } from './constants';
 
 /** The `SchemaUtils` class provides a wrapper around the publicly exported APIs in the `utils/schema` directory such
- * that one does not have to explicitly pass the `validator` or `rootSchema` to each method. Since both the `validator`
- * and `rootSchema` generally does not change across a `Form`, this allows for providing a simplified set of APIs to the
+ * that one does not have to explicitly pass the `validator`, `rootSchema`, or `defaultFormStateBehavior` to each method.
+ * Since these generally do not change across a `Form`, this allows for providing a simplified set of APIs to the
  * `@rjsf/core` components and the various themes as well. This class implements the `SchemaUtilsType` interface.
  */
 class SchemaUtils<T = any, S extends StrictRJSFSchema = RJSFSchema, F extends FormContextType = any>
   implements SchemaUtilsType<T, S, F>
 {
   rootSchema: S;
   validator: ValidatorType<T, S, F>;
+  behaviorBitFlags: number;
 
   /** Constructs the `SchemaUtils` instance with the given `validator` and `rootSchema` stored as instance variables
    *
    * @param validator - An implementation of the `ValidatorType` interface that will be forwarded to all the APIs
    * @param rootSchema - The root schema that will be forwarded to all the APIs
+   * @param behaviorBitFlags Bitwise flags to set which behavior is chosen for certain edge cases.
```

Comment:
```suggestion
   * @param [defaultFormStateBehavior] - Optional bit flag, if provided, allows users to override default form state behavior
```

Reply:
I'm open to discuss this but I had intentionally kept this parameter as required for the constructor here. Making the parameter for the `SchemaUtils` constructor makes it clear that this is important to the usage of `SchemaUtils`. Also, when I made it optional, it caused the unit tests to become more complex without providing much, if any, benefit. If a user wants to create a `SchemaUtils` instance I think it makes sense for them to explicitly specify which behavior they want. 

The only place this object is constructed is `createSchemaUtils()` in `createSchemaUtils.ts:289` and the parameter for that function is optional, which I believe accomplishes the same result from the perspective of the library's users.

Reply:
Oh, yeah I agree that making it required is necessary, I was just copy/pasting the name and documentation from other lines and missed this one.